contextutils/context_and_logging.go

```diff
@@ -114,6 +114,22 @@ func init() {
 	}
 }
 
+func BuildLoggerFromConfig(config zap.Config) (*zap.Logger, error) {
```

Comment:
Why are we only respecting certain configuration items? Shouldnt we just be adding to the existing background logger to make sure that things respect previously set settings?

This feels like a bandaid to a larger problem instead of addressing how we pull from background in a non-context dependant way

Reply:
The whole config is respected, as we return the output from `config.Build()` 

This covers a case [here](https://github.com/solo-io/dev-portal/pull/2893/files#diff-f36fcc143c1388c7c89b28b165cfb74d7f4c00fd6e58667a5594437f13180436L85) where we were previously creating a brand new logger directly from a `zap.Config`, but not incorporating the `LOG_TO_FILE_LOCATION` env

If we don't want to directly support creating a logger from config as part of this util we can find another path forward
There isn't a great reason not to other than that having the option may be confusing since in most cases it's preferable to use the logger from context

Reply:
Why arent we pulling from our fallback logger?
This seems like its still a mismatch in where one can configure logging and how it may or may not be respected